_bookdown.yml

```diff
@@ -11,9 +11,9 @@ rmd_files: ["index.Rmd",
             "01-introduction.Rmd",
             # Focus Topics
             "10-overview.Rmd",
-            "11-taxonomic-information.Rmd",
+            # "11-taxonomic-information.Rmd",
```

Comment:
These shouldn't be excluded from the book, should they?

Reply:
I had temporarily commented these out due to conversion problems, and forgot to uncomment. Yes, they shouldn't be excluded.